config/settings.yml

```
@@ -1,2 +1,16 @@
-help_page_url: "http://www.railstutorial.org/help"
-book_url: "http://www.railstutorial.org/book"
+defaults: &defaults
+  help_page_url: "http://www.railstutorial.org/help"
+  book_url: "http://www.railstutorial.org/book"
+
+  user:
+    MAX_NAME_LENGTH: 50
+    MAX_EMAIL_LENGTH: 255
```

```suggestion
    max_name_length: 50
    max_email_length: 255
```
trong file setting thì đặt tên biến bình thường là đc, biến hằng mới cần đặt viết hoa